draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -6450,7 +6460,8 @@ Expect: 100-continue
 </t>
 <t>
    A recipient &MUST; ignore the If-Unmodified-Since header field if the
-   received field value is not a valid HTTP-date.
+   received field value is not a valid HTTP-date, or if the field value has
+   more than one member.
 </t>
```

Comment:
This looks like a bug in the old spec. If-unmodified-since should behave like If-match and cause a 412 when it is invalid (because that doesn't match).

Reply:
Open a separate issue?

Reply:
On second thought (and seeing it has always been specified as ignore on invalid), let's just keep it that way.